content/applications/finance/accounting/fiscal_localizations/localizations/france.rst

```diff
@@ -5,21 +5,296 @@ France
+| 13 | Sens          | Can be "C" for Credit and "D" for Debit                                        | determines move_line.debit or move_line.credit      | Char                 |
++----+---------------+--------------------------------------------------------------------------------+-----------------------------------------------------+----------------------+
+
+Implementation details
+~~~~~~~~~~~~~~~~~~~~~~
+
+The following accounting entities are imported from the FEC files: **Accounts, Journals, Partners**,
+and **Moves**.
+
+Our module determines the encoding, the line-terminator character, and the separator that are used
+in the file.
+
+A check is then performed to see if every line has the correct number of fields corresponding to the
+header.
+
+If the check passes, then the file is read in full, kept in memory, and scanned. Accounting entities
+are imported one type at a time, in the following order.
+
+Accounts
+********
+
+Every accounting entry is related to an account, which should be determined by the field
+**CompteNum**.
+
+Code matching
+*************
+
+Should a similar account code already be present in the system, the existing one is used instead of
+creating a new one.
+
+Accounts in Odoo generally have a number of digits that are default for the fiscal localization. As
+the FEC module is related to the French localization, the default number of relevant digits is 6.
+
+This means that the account codes the trailing zeroes are right-trimmed, and that the comparison
+between the account codes in the FEC file and the ones already existing in Odoo is performed only on
+the first six digits of the codes.
+
+**Example**: the account code 65800000 in the file is matched against an existing 658000 account in
+Odoo, and that account is used instead of creating a new one.
+
+Reconcilable flag
+*****************
+
+An account is technically flagged as *reconcilable* if the first line in which it appears has the
+**EcritureLet** field filled out, as this flag means that the accounting entry is going to be
+reconciled with another one.
+
+.. note::
+   In case the line somehow has this field not filled out, but the entry still has to be reconciled
+   with a payment that hasn't yet been recorded, this isn't a problem anyway; the account is
+   flagged as reconcilable as soon as the import of the move lines requires it.
+
+Account type and Templates matching
+***********************************
+
+As the **type** of the account is not specified in the FEC format, **new** accounts are created
+with the default type *Current Assets* and then, at the end of the import process, they are
+matched against the installed Chart of Account templates. Also, the *reconcile* flag is also
+computed this way.
+
+The match is done with the left-most digits, starting by using all digits, then 3, then 2.
+
+**Example**::
```

Comment:
You could maybe try making use of the [Custom admonition](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#custom) directive and see if it looks better/clearer?

Reply:
I thought of this as well but there are other examples further that use a code that is not compatible with admonitions. It creates some kind of grey rectangle.

Is this a bug that should be fixed, @AntoineVDV ?

Reply:
I just tried that and it worked fine for me so I'm not sure what grey rectangle you're talking about?

FYI, I replaced all occurrences of `**Example**` by `.. Admonition:: Example`. If there were code-block markers (`::`), I kept them to avoid interpreting the example as RST code.

For example:
```rst
.. admonition:: Example

   ::

    Journal id = 5
    Moves:
        has a sale account line and no purchase account line = 0     ratio = 0
        has a purchase account line and no sale account line = 1     ratio = 0.25
        has a liquidity account line                         = 3     ratio = 0.75
                                                        ----------
                                                        Total: 4
```

Reply:
I've tidied up using a table.